File: vapor-app/src/main/java/com/tevinjeffrey/vapor/okcloudapp/model/CloudAppJsonItem.java
package com.tevinjeffrey.vapor.okcloudapp.model;

import com.google.gson.annotations.SerializedName;

public class CloudAppJsonItem {

    public Item item;
    public boolean deleted;

    public static class Item {
        public String name;
        @SerializedName("private")
        public boolean isPrivate;
        public String deletedAt;
        public String redirectUrl;
    }
}

File: vapor-app/src/main/java/com/tevinjeffrey/vapor/okcloudapp/model/CloudAppJsonItem.kt
package com.tevinjeffrey.vapor.okcloudapp.model

import com.google.gson.annotations.SerializedName

class CloudAppJsonItem {

    var item: Item? = null
    var deleted: Boolean = false

    class Item(name: String? = null, isPrivate: Boolean = false, deleteAt: String? = null, redirectUrl: String?  = null) {
        var name: String? = null
        @SerializedName("private")
        var isPrivate: Boolean = false
        var deletedAt: String? = null
        var redirectUrl: String? = null
    }
}
